WalletWasabi/Helpers/Constants.cs

```diff
@@ -22,7 +22,10 @@ public static class Constants
 	public const int P2wpkhInputSizeInBytes = 41;
 	public const int P2wpkhInputVirtualSize = 69;
 	public const int P2pkhInputSizeInBytes = 145;
+	public const int P2pkhOutputVirtualSize = 34;
```

Comment:
payment batching means we may pay to various outputs and so we need to take their output size into consideration when registering them.